packages/arb-token-bridge-ui/src/components/TransferPanel/TokenImportDialog.tsx

```diff
@@ -300,12 +299,6 @@ export function TokenImportDialog({
       })
     }
 
-    // do not allow import of withdraw-only tokens at deposit mode
-    if (isDepositMode && isWithdrawOnlyToken(l1Address, childChain.id)) {
-      openTransferDisabledDialog()
-      return
-    }
```

Comment:
not needed because after importing, the tx disabled dialog will open after completing the async fetch if it's a LayerZero token / on the withdraw-only list